ext/date/php_date.c

```diff
@@ -4523,34 +4535,33 @@ static void php_do_date_sunrise_sunset(INTERNAL_FUNCTION_PARAMETERS, int calc_su
 	ZEND_PARSE_PARAMETERS_START(1, 6)
 		Z_PARAM_LONG(time)
 		Z_PARAM_OPTIONAL
-		Z_PARAM_LONG(retformat)
-		Z_PARAM_DOUBLE(latitude)
-		Z_PARAM_DOUBLE(longitude)
-		Z_PARAM_DOUBLE(zenith)
-		Z_PARAM_DOUBLE(gmt_offset)
+		Z_PARAM_LONG_OR_NULL(retformat, retformat_is_null)
```

Comment:
This one has SUNFUNCS_RET_STRING as a proper default, so it shouldn't be nullable.